embedded-graphics/CHANGELOG.md

```diff
@@ -0,0 +1,406 @@
+# Changelog
+
+Embedded Graphics is a `no_std` library for adding graphics features to display drivers. It aims to use the minimum amount of memory for builtin graphics objects by leveraging Rust's iterators to avoid large allocations. It targets embedded environments, but can run anywhere like a Raspberry Pi up to full desktop machines.
```

Comment:
It can't hurt? It's handy for people directly visiting the changelog to know what the project is about, similar to Rust announcement blog posts where there's a "Rust is a ..." preamble.

Reply:
No, it can't really hurt, but it is one more description that needs to be maintained. And I think it's uncommon to include this in a changelog (but I might be wrong).

IMO an announcement blog post is quite different from a changelog. Do you think many people will read the changelog before knowing anything about the project?